cmd/bif/extract.go

```diff
@@ -0,0 +1,97 @@
+// SPDX-FileCopyrightText: © 2025 SBT Localization https://sbt.localization.com.ua
+// SPDX-FileContributor: @definitelythehuman
+// SPDX-FileContributor: Serhii Olendarenko <sergey.olendarenko@gmail.com>
+//
+// SPDX-License-Identifier: GPL-3.0-only
+
+package bif
+
+import (
+	"io"
+	"log"
+	"os"
+	"path/filepath"
+
+	"github.com/sbtlocalization/sbt-infinity/config"
+	"github.com/sbtlocalization/sbt-infinity/fs"
+	"github.com/spf13/cobra"
+)
+
+func NewExportCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "extract -k path-to-chitin.key [-o output-dir][-t resource-type][-f regex-filter]",
```

Comment:
I'd make it shorter:
```suggestion
		Use:     "extract [-o output-dir] [-t type]... [-f regex]",
```